### DISCUSSION TITLE
how to use --listen and --port arguments in alltalk tts standalone?

### DISCUSSION THREAD

**Discussion by aplayz522** [erew123/alltalk_tts#327] [Q&A] [answered]
Hi guys,

Im trying to find out how to use --listen and --port arguments in the standalone version of alltalk tts. i didnt find anything about it online but I edited the start_alltalk.bat file by adding --listen and --port to when it calls the launcher script but it hasnt worked. Does anyone know a solution?

**Comment by erew123** [comment, marked as answer]
v1 of AllTalk https://github.com/erew123/alltalk_tts?tab=readme-ov-file#-changing-alltalks-ip-address--accessing-alltalk-over-your-network

v2 Binds to 0.0.0.0 (so all IP addresses) and you can edit the port in the Gradio UI under the [Global settings](https://github.com/erew123/alltalk_tts/discussions/237)

**Reply by aplayz522** [reply]
thanks!
